website/docs/cdktf/examples.mdx

```diff
@@ -26,72 +26,76 @@ Follow these hands-on tutorials from HashiCorp Learn:
-| [azure](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/azure)                                  | Provisions a Virtual Network on Microsoft Azure.                                                                                                                                                 |
-| [azure-app-service](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/azure-app-service)          | Uses the Azure App Service to deploy a Docker image.                                                                                                                                             |
-| [docker](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/docker)                                | Starts a Docker container with an NGINX server.                                                                                                                                                  |
-| [google](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/google)                                | Creates a simple Compute Instance with the Google Cloud Platform Provider.                                                                                                                       |
-| [google-cloudrun](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/google-cloudrun)              | Defines a Google Cloud Run service with a hello world container.                                                                                                                                 |
-| [kubernetes](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/kubernetes)                        | Sets up a Kubernetes Deployment with a NGINX container.                                                                                                                                          |
-| [ucloud](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/ucloud)                                | Provisions a Linux base image on UCloud.                                                                                                                                                         |
-| [vault](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/vault)                                  | Creates a Mount in Vault.                                                                                                                                                                        |
+| Example                                                                                                                  | Description                                                                                                                                                                                      | Size  |
```

Comment:
I like this and the way that you've changed the names to have "end-to-end"! 

Instead of size, I wonder if instead we should say "Complexity" and say that it's either High or Low. I wonder if that will communicate the meaning a little bit better than size (which I guess works, but I feel like maybe complexity is more precise?)